config/server/consumer.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const { env } = process;
+
+module.exports = {
+  webhookUrl: env.CONSUMER_WEBHOOK_URL,
+  clientID: env.CONSUMER_CLIENT_ID,
```

Comment:
I believe the convention here would imply camelcase:
```suggestion
  clientId: env.CONSUMER_CLIENT_ID,
```
You wouldn't need to alias it when importing [here](https://github.com/ExtensionEngine/tailor/pull/702/files#diff-7efff118cb81130382f12499a92b5d65874901ddd644818c97a1cfc3596cee5aR4)